plugins/jobs/src/js/jobsToggleSchedule.js

```diff
@@ -0,0 +1,22 @@
+import { updateSchedule } from "#SRC/js/events/MetronomeClient";
+
+export default function jobsToggleSchedule(job) {
+  const [schedule] = job.schedules;
+  const isEnabled = schedule ? schedule.enabled : false;
+
+  const label = isEnabled ? "Disable Schedule" : "Enable Schedule";
+
+  const data = Object.assign({}, schedule, { enabled: !isEnabled });
+
+  return {
+    label,
+    onItemSelect() {
+      // TODO - use graphql here https://jira.mesosphere.com/browse/DCOS-37617
+      // take(1) makes sure the observable is going to complete after finishing
+      // the request, so we don't have to care about unsubscribing.
+      updateSchedule(job.id, data)
+        .take(1)
```

Comment:
I learned that there's also `first()` :)

Reply:
🎨 +1 `first()`

Reply:
Hm, the blogpost is referencing to RxJS v4, is it the same in 5 (and 6)? 

Also api says something else:
http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-first

And `Rx.Observable.of([]).first()` does not thow… 🤔

Reply:
On rxjs 5, the version we use, it says: `EmptyError | Delivers an EmptyError to the Observer's error callback if the Observable completes before any next notification was sent.`